scripts/dependency_versions.sh

```diff
@@ -5,9 +5,12 @@ DATAFED_GLOBUS_VERSION="6.0.22"
 DATAFED_JSON_SCHEMA_VALIDATOR_VERSION="2.1.0"
 DATAFED_NLOHMANN_JSON_VERSION="3.10.2"
 DATAFED_LIBSODIUM_VERSION="1.0.18"
+DATAFED_LIB_LIBSODIUM_VERSION="23.3.0"
```

Comment:
LIB_LIB needs to be documented somewhere preferrably in this file. How is this different from just LIB.